pkgs/development/interpreters/python/cpython/default.nix

```diff
@@ -99,31 +100,6 @@ in with passthru; stdenv.mkDerivation {
     # (since it will do a futile invocation of gcc (!) to find
     # libuuid, slowing down program startup a lot).
     (./. + "/${sourceVersion.major}.${sourceVersion.minor}/no-ldconfig.patch")
-  ] ++ optionals stdenv.isLinux [
-    # Optimize symbol tables for the sake of dynamic linking.
-    # Significant for Python because of extension modules.
-    (
-      if pythonAtLeast "3.8" then
-        fetchpatch {
-          url = "https://salsa.debian.org/cpython-team/python3/-/raw/3.8.3rc1-1/debian/patches/link-opt.diff";
```

Comment:
All of these are files in git, not generated files so `fetchurl` should be sufficient.

Reply:
yes, it *should* be, yet ofborg complains.

Reply:
I see the error even on a337c44 and it points to the removed line :woman_shrugging: